stdlib/gc.mli

```diff
@@ -410,20 +410,20 @@ val delete_alarm : alarm -> unit
 (** [delete_alarm a] will stop the calls to the function associated
    to [a]. Calling [delete_alarm a] again has no effect. *)
 
-external eventlog_pause : unit -> unit = "caml_eventlog_pause"
-(** [eventlog_pause ()] will pause the collection of traces in the
+external eventring_pause : unit -> unit = "caml_eventring_pause"
+(** [eventring_pause ()] will pause the collection of traces in the
    runtime.
    Traces are collected if the program is linked to the instrumented runtime
    and started with the environment variable OCAML_EVENTLOG_ENABLED.
```

Comment:
Also, historically, the `OCAML_EVENTLOG_PREFIX` API was required because the Multicore runtime would eventually need more than one CTF stream (one per domain.).
Thus this API takes the prefix of the many files that would be generated by eventlog.
I think with the current eventring design we could get away with just a `_FILE` variable, pointing to a single file. (which I think is what your implementation does: we can make it more explicit.)

Reply:
Yes, I agree. I've removed the eventring_ functions from `Gc` and because they're on `Eventring` anyway. Also the environment variable is `OCAML_EVENTRING_START` to enable it from startup.

I'm going to change `OCAML_EVENTRING_PREFIX` to `OCAML_EVENTRING_DIR` because it semantically different. I think it needs to be a directory rather than a file because otherwise we end up with problems if people fork.